tests/purs/failing/FoldableInstance9.out

```diff
@@ -215,10 +215,10 @@ Error 9 of 68:
 
     The type variable `a` must only be used as the last argument in a data type:
 
-      tests/purs/failing/FoldableInstance9.purs:85:29 - 85:30 (line 85, column 29 - line 85, column 30)
+      tests/purs/failing/FoldableInstance9.purs:86:24 - 86:25 (line 86, column 24 - line 86, column 25)
 
 
-  under the label `[33mrights[0m`
+  under the label `[33mlefts[0m`
```

Comment:
Could these source spans be reordered so that what appears in source code first is what's displayed as an error first? Right now, this is sorting the errors by the alphabetical ordering of the labels.

Reply:
Sure, that's reasonable.